pyproject.toml

```
@@ -0,0 +1,16 @@
+[build-system]
+requires = ["setuptools>=64.0", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "geo-calculator"
+authors = [{ name = "Victoria Myklebust Olsen" }]
+description = "Introduction to Software Development course and demo Python package"
+readme = "README.md"
+requires-python = ">=3.13"
+license = { file = "LICENSE" }
+dependencies = ["numpy"]
+dynamic = ["version"]
+
+[project.urls]
+repository = "https://github.com/<Victoriamyklebustolsen>/intro-to-software-development"
```

The workflow can be fixed by adding the dev dependency

```suggestion
repository = "https://github.com/<Victoriamyklebustolsen>/intro-to-software-development"

[project.optional-dependencies]
dev = ["pytest"]
```